experiment-runner/ConfigValidator/Config/Validation/ConfigValidator.py

```diff
@@ -27,6 +70,13 @@ def validate_config(config: RunnerConfig):
         if '~' in str(config.experiment_path):
             config.experiment_path = config.experiment_path.expanduser()
         
+        # Set defaults to support configs without the self_measure parameter
+        if not hasattr(config, "self_measure"):
+            config.self_measure = False
+
+        if config.self_measure and not hasattr(config, "self_measure_bin"):
+            config.self_measure_bin = "/usr/local/bin/energibridge" # This is spesific to linux, might work for osx as well
```

Comment:
Apparently the way to make it runnable from anywhere on mac is to `export PATH` (on the newer `zsh` terminals)

Reply:
I assumed that mac had a PATH env, I wasn't sure if it contained /usr/local/bin (where users on linux put their non-system installed binaries). Sadly I don't have one to test it on :(